templates/windows/chocolateyUninstall.ps1.erb

```diff
@@ -0,0 +1,20 @@
+$toolsDir = "$(Split-Path -parent $MyInvocation.MyCommand.Definition)"
+
+$contentFile=(join-path $toolsDir 'file-list') + ".txt"
+if ((Test-Path -path $contentFile)) {
+  Write-Debug "Removing files defiend in `'$contentFile`'"
+  $contentFile
+  $contents=get-content $contentFile
+  foreach ($fileInstalled in $contents) {
+    $fileInstalled.
```

Comment:
no idea, copied it from https://github.com/chocolatey/choco/blob/master/src/chocolatey.resources/helpers/functions/UnInstall-ChocolateyZipPackage.ps1#L50 though the `.` is a typo